### ISSUE TITLE
ArkInventory-3.09.32-Classic "GetBackpackCurrencyInfo" nil value

### ISSUE THREAD

**Issue by speedwaystar** [arkayenro/arkinventory#1344]
presumably GetBackpackCurrencyInfo is a retail-only feature.

```
2x ArkInventory\ArkInventory-30932.lua:12187: ArkInventory\ArkInventoryClient.lua:192: attempt to call global 'GetBackpackCurrencyInfo' (a nil value)
[C]: ?
ArkInventory\ArkInventory-30932.lua:12187: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>

Locals:
(*temporary) = "ArkInventory\ArkInventoryClient.lua:192: attempt to call global 'GetBackpackCurrencyInfo' (a nil value)"
```

**Comment by arkayenro** [comment]
could you try [r1007-alpha](https://www.curseforge.com/wow/addons/ark-inventory/files/3044913) and let me know if that fixes it

**Comment by vutruzec** [comment]
r1007-alpha is working fine for me, not getting the error anymore, thanks

**Comment by arkayenro** [comment]
fixed in 3.09.33
